backend/populateDB/data.sql

```diff
@@ -26,17 +24,16 @@ select floods.register_user(text 'Community', text 'Editor Too', text 'Community
 alter sequence floods.user_id_seq restart with 6;
 select floods.register_user(text 'Inactive', text 'User', text 'Retired', integer '1', text '867-5309', text 'inactive@flo.ods', text 'texasfloods', text 'floods_community_editor');
 
-
 -- Add crossings
-insert into floods.crossing (id, name, human_address, description, coordinates) values
-  (1, 'park', 'at the park', 'Crossing at the park', ST_MakePoint(-97.768, 30.267)),
-  (2, 'school', 'at the school', 'Crossing at the school', ST_MakePoint(-97.768, 30.267)),
-  (3, 'library', 'at the library', 'Crossing at the library', ST_MakePoint(-97.768, 30.267)),
-  (4, 'capitol', 'at the capitol', 'Crossing at the capitol', ST_MakePoint(-97.768, 30.267)),
-  (5, 'city hall', 'at city hall', 'Crossing at city hall', ST_MakePoint(-97.768, 30.267)),
-  (6, 'coffee shop', 'at the coffee shop', 'Crossing at the coffee shop', ST_MakePoint(-97.768, 30.267)),
-  (7, 'other community', 'in the other community', 'Crossing in the other community', ST_MakePoint(-97.768, 30.267)),
-  (8, 'other community 2', 'another in the other community', 'Another crossing in the other community', ST_MakePoint(-97.768, 30.267));
+insert into floods.crossing (id, name, human_address, description, coordinates, geojson) values
```

Comment:
I think this makes sense, but why do crossings need both a coordinates and geojson? It looks like the geojson is just a point saved is a specific format.

Reply:
I tried using a function to calculate out the geojson instead of storing it, but the queries were taking FOREVER. We still want to have the postgis object to do calculations (such as getting a viewport that contains all crossings) within Postgres, but calculating on write is much faster on read than calculating on read.